Packs/Endace/Integrations/Endace/Endace.yml

```diff
@@ -0,0 +1,383 @@
+category: Network Security
+commonfields:
+  id: Endace
+  version: -1
+configuration:
+- additionalinfo: FQDN/IP address of EndaceProbe or Endace Investigation Manager
+  display: EndaceProbe URL (e.g. https://endaceprobe.com)
+  name: applianceurl
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
+description: "The EndaceProbe Analytics Platform provides 100% accurate, continuous\
+  \ packet capture on network links up to 100Gbps, with unparalleled depth of storage\
+  \ and retrieval performance.  Coupled with the Endace InvestigationManager, this\
+  \ provides a central search and data-mining capability across a fabric of EndaceProbes\
+  \ deployed in a network. \n                                     \nThis integration\
+  \ uses Endace APIs to search, archive and download PCAP file from either a single\
+  \ EndaceProbe or many via the InvestigationManager and enables integration of full\
+  \ historical packet capture into security automation workflows."
+display: Endace
+name: Endace
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Event Start Time in ISO 8601 format as in 2020-04-08T15:46:30
+      isArray: false
+      name: start
+      required: false
+      secret: false
+    - default: false
+      description: Event End Time in ISO 8601 format  as in 2020-04-08T15:46:30
+      isArray: false
+      name: end
+      required: false
+      secret: false
+    - default: false
+      description: List of Source IP addresses to search with a maximum of 10 IP addresses
+        per search. For valid search either a Src Host or a Dest Host value is required.
+      isArray: true
+      name: src_host_list
+      required: false
+      secret: false
+    - default: false
+      description: List of Destination IP addresses to search with a maximum of 10
+        IP addresses per search. For valid search either a Src Host or a Dest Host
+        value is required.
+      isArray: true
+      name: dest_host_list
+      required: false
+      secret: false
+    - default: false
+      description: List of Source Port addresses to search with a maximum of 10 Port
+        addresses per search.
+      isArray: true
+      name: src_port_list
+      required: false
+      secret: false
+    - default: false
+      description: List of Destination Port addresses to search with a maximum of
+        10 Port addresses per search.
+      isArray: true
+      name: dest_port_list
+      required: false
+      secret: false
+    - default: false
+      description: TCP or UDP

```

Comment:
please add predefined values here

Reply:
@Itay4 : There is no predefined value. it can be either a string or number as defined by IANA.

Reply:
so the description is not clear enough, please improve it